fulp_modules/features/antagonists/bloodsuckers/code/powers/_powers.dm

```diff
@@ -53,6 +53,9 @@
 	bloodsuckerdatum_power = null
 	return ..()
 
+/datum/action/cooldown/bloodsucker/IsAvailable(feedback = FALSE)
+	return next_use_time <= world.time
```

Comment:
what does parent do here?

Reply:
same thing but we don't call parents cause checks are done on can_use